src/Symfony/Component/Messenger/Stamp/OverrideSendersStamp.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Stamp;
+
+/**
+ * Apply this stamp to change sender to be used instead of the one from configuration file.
+ */
+final class ViaSenderStamp implements StampInterface
```

Comment:
The class name doesn't match the filename anymore

Reply:
any opinion about the best name `ViaSenderStamp`? `SenderNamesStamp`? (`OverrideSendersStamp` wouldn't fit if we go with [my proposal](https://github.com/symfony/symfony/pull/39306#issuecomment-1084588686) to not allow senders to be set via both config and stamps)

Reply:
`SenderNamesStamp` seems more consistent with `BusNameStamp`